docs/src/man/structure.md

```diff
@@ -244,6 +232,210 @@ julia> rad2deg(psiangle(struc['A'][50], struc['A'][51]))
+```
+
+
+## RCSB PDB Utility Functions
+
+- To download the entire RCSB PDB database in your preferred file format:
+
+    ```julia
+    downloadentirepdb(pdb_dir="path/to/pdb/directory/", file_format=MMTF, overwrite=false)
+    ```
+
+    The keyword arguments are described below:
+
+| Keyword Argument                 | Description                                                                                              |
+| :------------------------------- | :------------------------------------------------------------------------------------------------------- |
+| `pdb_dir::AbstractString=pwd()`  | The directory to which the PDB files are downloaded                                                      |
+| `file_format::Type=PDB`          | The format of the PDB file. Options are PDB, PDBXML, MMCIF or MMTF                                       |
+| `overwrite::Bool=false`          | If set `true`, overwrites the PDB file if exists in `pdb_dir`; by default skips downloading the PDB file |
+
+- To update your local PDB directory based on the weekly status list of new, modified and obsolete PDB files from RCSB Server:
+
+    ```julia
+    updatelocalpdb(pdb_dir="path/to/pdb/directory/", file_format=MMTF)
+    ```
+
+    The `file_format` specifies the format of the PDB files present in the local PDB directory. Obsolete PDB files are stored in the autogenerated `obsolete` directory inside the specified local PDB directory.
+
+- To download all obsolete PDB files from RCSB Server:
+
+    ```julia
+    downloadallobsoletepdb(;obsolete_dir="/path/to/obsolete/directory/", file_format=MMCIF, overwrite=false)
+    ```
+
+    The `file_format` specfies the format in which the PDB files are downloaded; Options are PDB, PDBXML, MMCIF or MMTF.
+
+    If `overwrite=true`, the existing PDB files in obsolete directory will be overwritten by the newly downloaded ones.
+
+There are a few more functions that may help.
+
+| Function           | Returns                                                                         | Return type                                              |
+| :----------------- | :------------------------------------------------------------------------------ | :------------------------------------------------------- |
+| `pdbentrylist`     | List of all PDB entries from RCSB Server                                        | `Array{String,1}`                                        |
+| `pdbstatuslist`    | List of PDB entries from specified RCSB weekly status list URL                  | `Array{String,1}`                                        |
+| `pdbrecentchanges` | Added, modified and obsolete PDB lists from the recent RCSB weekly status files | `Tuple{Array{String,1},Array{String,1},Array{String,1}}` |
+| `pdbobsoletelist`  | List of all obsolete PDB entries in the RCSB server                             | `Array{String,1}`                                        |
+
+
+## Maintaining a local copy of the entire RCSB PDB Database
```

Comment:
This section title is quite long, and the section mainly refers to the above section. Perhaps remove this section and put the sentence "Run the `downloadentirepdb` function..." into the above section.